source/js/nav-newsletter.js

```diff
@@ -69,17 +76,24 @@ class NavNewsletter {
   // remove the global 'closeFormClickHandler' click event handler,
   // and reset the form
   closeMobileNewsletter() {
+    const wrapper = elements.container;
     elements.narrowMenuContainer.classList.remove("d-none");
-    elements.container.classList.remove("faded-in");
+    wrapper.classList.remove("faded-in");
+    setTimeout(() => {
```

Comment:
Can we please have some inline comments here as well?

Reply:
yep!

Reply:
```suggestion
    // Schedule a "display:none" to happen after the `expanded` animation finishes.
    // See `#nav-newsletter-form-wrapper` transition in ./source/sass/components/primary-nav.scss 
    setTimeout(() => {
```